### PR TITLE
Leah/create invite screen

### PR DESCRIPTION
**Pull request by Leah-Xia-Microsoft**
## Purpose
- Layout InviteScreen

- User Story: As a user, I can invite others to join my call

- Design: 

- Current Implementation:

## Does this introduce a breaking change?

```
[x] Yes
[ ] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[x] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
Run and test the code

## What to Check
Layout

### CODE REVIEW

File: AzureCalling/Controllers/InviteViewController.swift [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+        continueButton.expandHorizontallyInSuperView(withEqualMargin: 16)
+        continueButton.pinToBottom(withMargin: 16)
+    }
+
+    private func layoutMainContainer() {
+        let mainContainer = UIView()
+        view.addSubview(mainContainer)
+        mainContainer.expandHorizontallyInSuperView(withEqualMargin: 16)
+        mainContainer.pinToTop(withMargin: 0)
+        mainContainer.bottomAnchor.constraint(equalTo: continueButton.topAnchor, constant: -12).isActive = true
```
Comment (emlynmac) [inline AzureCalling/Controllers/InviteViewController.swift:79]: A minor thing, but IMHO having positive constants is easier to understand, which can be done by reversing the anchors

File: AzureCalling/Controllers/InviteViewController.swift [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+
+        let iconContainer = PaddingContainer(containing: iconImageView, padding: UIEdgeInsets(top: 29, left: 0, bottom: 4, right: 0))
+        iconImageView.fixHeightTo(height: 28)
+
+        let buttonContainer = PaddingContainer(containing: shareButton, padding: UIEdgeInsets(top: 32, left: 76.5, bottom: 12, right: 76.5))
```
Comment (emlynmac) [inline AzureCalling/Controllers/InviteViewController.swift:84]: Half-point layout is not usually a good idea; on 3x screens, anti-aliasing will occur.

File: AzureCalling/Controllers/StartCallViewController.swift [hunk 3]
```diff
@@ -42,6 +42,11 @@ class StartCallViewController: UIViewController {
         view.addGestureRecognizer(tap)
     }
 
+    override func viewWillDisappear(_ animated: Bool) {
+        super.viewWillDisappear(animated)
+        navigationItem.backBarButtonItem = UIBarButtonItem(title: "", style: .plain, target: nil, action: nil)
```
Comment (kevinyulu) [inline AzureCalling/Controllers/StartCallViewController.swift:47]: Is setting the title to hide the back button? if yes, we might also try navigationItem.backBarButtonItem to nil.

**Review by kevinyulu — APPROVED** [review]
tested in iPhone Simulator lgtm

**Review by AmyL219 — APPROVED** [review]

**Pull request merged**